src/Sql/Auth/dbo/Tables/Grant.sql

```diff
@@ -1,6 +1,6 @@
 ﻿CREATE TABLE [dbo].[Grant]
 (
-    [Id] INT NOT NULL IDENTITY,
+    [Id] INT NOT NULL IDENTITY(1,1),
```

Comment:
ℹ️ Same as the above -- MSSQL handles this automatically and it's implied. This migration worked fine for us in our cloud.